### PR TITLE
feat(HACBS-1150): fix apiexport for infra-deployments

### PR DESCRIPTION
**Pull request by hongweiliu17**
Signed-off-by: Hongwei Liu <hongliu@redhat.com>

### CODE REVIEW

**Comment by dirgim** [comment]
@hongweiliu17 I think that running `make manifests` should resolve the failing Check sources CI test.

**Review by dirgim — APPROVED** [review]
LGTM

**Comment by hongweiliu17** [comment]
> @hongweiliu17 I think that running `make manifests` should resolve the failing Check sources CI test.

Thank you! I didn't know how to fix it.

**Review by amisstea — APPROVED** [review]
LGTM :+1:

File: .tekton/push.yaml [hunk 1]
```diff
@@ -21,7 +21,8 @@ spec:
       value: Dockerfile
     - name: infra-deployment-update-script
       value: |
-        sed -i -e 's|\(https://github.com/redhat-appstudio/integration-service/config/default?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/integration/kustomization.yaml
+        sed -i -e 's|\(https://github.com/redhat-appstudio/integration-service/.*?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/application-service/base/kustomization.yaml
+        ./hack/generate-apiexport-overlays.sh components/hacbs/application-service/
```
Comment (Michkov) [inline .tekton/push.yaml:25]: application-service -> integration-service

Comment (hongweiliu17) [reply .tekton/push.yaml]: yes...

**Review by jsztuka — APPROVED** [review]
lgtm

**Pull request merged**